dashboard-widgets.el

```diff
@@ -251,12 +251,6 @@ Example:
   :type '(function string)
   :group 'dashboard)
 
-(defcustom dashboard-footer
-  (nth (random (1- (1+ (length dashboard-footer-messages)))) dashboard-footer-messages)
-  "A footer with some short message."
-  :type 'string
-  :group 'dashboard)
```

Comment:
Why are we removing the random footer? 🤔

Reply:
We are not using that custom variable. The `dashboard-insert-footer` function uses `dashboard-random-footer` to get the value.  The `dashboard-footer-message` list is used to customize the text, from the README:
```emacs-lisp
(setq dashboard-footer-messages '("Dashboard is pretty cool!"))
```

Reply:
Good catch! Now sure why we still have this variable in the first place. 🤔